torchreid/utils/tools.py

```diff
@@ -168,6 +168,14 @@ def get_model_attr(model, attr):
     return getattr(model, attr)
 
 
+def set_model_attr(model, attr, value):
```

Comment:
Please, use set_model_attr everywhere, for example there: https://github.com/openvinotoolkit/deep-object-reid/blob/481396ce0dcb009b04c6232ad1d982d66a9a2858/torchreid/integration/sc/inference_task.py#L280

Reply:
Will fix that in another PR.